ocdstoucan/settings.py

```diff
@@ -168,3 +168,5 @@
         dsn=os.getenv('SENTRY_DSN'),
         integrations=[DjangoIntegration(), SqlalchemyIntegration()]
     )
+
+OCDS_TOUCAN_CREDENTIALS_DRIVE = os.getenv('OCDS_TOUCAN_CREDENTIALS_DRIVE', 'credentials.json')
```

Comment:
Redirects us to this page instead of the google login page:

![imagen](https://user-images.githubusercontent.com/48607824/81593370-58efbe00-938d-11ea-9f22-28c8aff11bbf.png)

Reply:
Can you add a comment with a link to the guide from the issue description?

Reply:
Yes! https://github.com/open-contracting/toucan/issues/42#issuecomment-631126853